<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saf</class>
 <widget class="QMainWindow" name="saf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>saf</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>561</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="size">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>100</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="size_add">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="size_del">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="size_b">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Задать</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="size_n">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="fill">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="fill_s">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Нули</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Случ.</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="fill_b">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Заполнить</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="sort">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="sort_b">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Сорт.</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sort_s">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>БЫСТР</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ОБЕЗЬЯН.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>РАСЧЕС.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ПУЗЫРЕК</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ГНОМ</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="del_dub">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Удл. дубликат</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="find">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="find_s">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Лин.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Бин.</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="find_b">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="find_n">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="status">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="value_status">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sort_status">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Не сорт</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="sort_error">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>330</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="db_error">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="find_o">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fill_error">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="kostil">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
